e2e/cypress/e2e/spec.cy.js

```diff
@@ -227,4 +230,116 @@ describe("journey test", () => {
       });
     });
   });
+
+  // Testing Recurring events
+
+  describe("recurring events", () => {
+    it.only("displays monthly recurring events", () => {
```

Comment:
Forgot an `.only`

Reply:
If there's something getting in the way of a clean test run, let's figure that out before merge so we can remove the .only and run the full suite of tests.